rllib/utils/schedules/schedule.py

```diff
@@ -26,19 +29,26 @@ def __init__(self, framework=None):
         self.framework = check_framework(framework)
 
     @abstractmethod
-    def value(self, t):
+    def _value(self, t):
         """
-        Returns the value based on a time value.
+        Returns the value based on a time step input.
 
         Args:
-            t (int): The time value (e.g. a time step).
-                NOTE: This could be a tf.Tensor.
+            t (int): The time step. This could be a tf.Tensor.
 
         Returns:
             any: The calculated value depending on the schedule and `t`.
         """
         raise NotImplementedError
 
+    def value(self, t):
```

Comment:
I think we can get rid of self.framework variable by renaming this call to .tf_value right? That would also be more explicit.

Reply:
Could we leave it as is for now? There are a lot of possible drawbacks to this split method approach. Once we are done with the new Exploration classes, we can revisit this question and see what works and what doesn't.

Reply:
RLlib tests all seem ok.